x-pack/test/functional/services/ml/common_ui.ts

```diff
@@ -163,5 +163,54 @@ export function MachineLearningCommonUIProvider({ getService }: FtrProviderConte
       // escape popover
       await browser.pressKeys(browser.keys.ESCAPE);
     },
+
+    async setSliderValue(testDataSubj: string, value: number) {
```

Comment:
Great to have this available here so it's re-usable! Could we let the `data_frame_analytics_creation` service methods `setTrainingPercent` and `assertTrainingPercentValue` use this implementation, so we avoid code duplication?

Reply:
Refactored in 2b161425455c1cbfb5c247842fb69df464951435

Reply:
Could we do the same for `data_frame_analytics_creation.assertTrainingPercentValue` -> `common_ui.assertSliderValue`? This is not only used to check that `setTrainingPercent` worked correctly, but also during clone to check that the slider value is carried over correctly.